### DISCUSSION TITLE
[Code Reviews] ⏰💤 Snooze Comments for Later

### DISCUSSION THREAD

**Discussion by cpholguera** [community/community#10615] [Mobile] [answered]
Sometimes I receive a code review and there are some comments (most of them usually suggested changes) that can be applied quickly on mobile. Some others will need more though/discussion or I will have to implement the changes on my laptop and push them.

It would be great to have a way to postpone or snooze them. Or "Save for Later".Some kind of marker, label that can be later clicked to make those comments reaper. Of course it shouldn't be possible to merge unless there are no such "postponed comments" and a warning must pop up in that case.

This will make the reviewing experience much cleaner, the comments you cannot process on that moment won't pollute the rest of the review process you would like to continue doing in the go.

Of course this will also be great for desktop but I think mobile is the place where it's most helpful right now.

Thank you!

**Comment by brianlovin** [comment, marked as answer]
Thanks for the detailed suggestion @cpholguera — I think I understand the flow you're picturing and how that'd be useful. I'll pass this along to our team, but I can say now that this flow isn't on our roadmap for the near future. Looking forward to seeing if more comments and requests for this come in, and that can help us prioritize!

**Reply by cpholguera** [reply]
Then I'm kind of glad to see that it's a brand new idea no one thought about before 😁 I hope your team likes it and let's see if other people here think it's also interesting for them. Thank you @brianlovin!

**Comment by lordt4ever** [comment]
Me
